From: Lars Kellogg-Stedman <lars@oddbit.com>
From: Jakub Jirutka <jakub@jirutka.cz>
Date: Fri, 19 Aug 2022 22:32:10 -0400
Subject: [PATCH 7/8] Use correct extension for slapd modules

The example slapd.conf and slapd.ldif provided by this package attempts
to load modules with a .la suffix, but the package only provides .so
modules.

Update the example slapd.conf to use the correct extension.

See https://gitlab.alpinelinux.org/alpine/aports/-/issues/13866

diff --git a/servers/slapd/slapd.conf b/servers/slapd/slapd.conf
--- a/servers/slapd/slapd.conf
+++ b/servers/slapd/slapd.conf
@@ -15,8 +15,8 @@ argsfile	%LOCALSTATEDIR%/run/slapd.args
 
 # Load dynamic backend modules:
 modulepath	%MODULEDIR%
-moduleload	back_mdb.la
-# moduleload	back_ldap.la
+moduleload	back_mdb.so
+# moduleload	back_ldap.so
 
 # Sample security restrictions
 #	Require integrity protection (prevent hijacking)

diff --git a/servers/slapd/slapd.ldif b/servers/slapd/slapd.ldif
--- a/servers/slapd/slapd.ldif
+++ b/servers/slapd/slapd.ldif
@@ -30,9 +30,9 @@
 objectClass: olcModuleList
 cn: module
 olcModulepath:	%MODULEDIR%
-olcModuleload:	back_mdb.la
-#olcModuleload:	back_ldap.la
-#olcModuleload:	back_passwd.la
+olcModuleload:	back_mdb.so
+#olcModuleload:	back_ldap.so
+#olcModuleload:	back_passwd.so
 
 dn: cn=schema,cn=config
 objectClass: olcSchemaConfig
